tests/e2e/deposit.test.js

```diff
@@ -0,0 +1,75 @@
+const depositViewButtonClick = async (page) => {
+  const depositViewButton = await page.getByTestId('open-deposit-view-button', true);
+  await depositViewButton.click();
+};
+
+const checkIsCopyButtonTextMatch = async (page, name, id) => {
+  const copyButton = await page.getByTestId(`copy-${name}-button`, true);
+  await copyButton.click();
+  const copiedText = await page.evaluate(() => navigator.clipboard.readText());
+  expect(copiedText).toBe(id);
+};
+
+const qrCodeIconButtonClick = async (page) => {
+  const qrCodeIconButton = await page.getByTestId('qr-code-icon-button', true);
+  await qrCodeIconButton.click();
+};
+
+const closeQrCodeModalButtonClick = async (page) => {
+  const qrCodeIconButton = await page.getByTestId('close-qr-code-modal-button', true);
+  await qrCodeIconButton.click();
+};
+
+const checkIsSubtitleTextMatch = async (page, name, text) => {
+  const subtitle = await page.getByTestId(`deposit-${name}-subtitle`, true);
+  const subtitleText = await page.evaluate((el) => el.textContent, subtitle);
+  expect(subtitleText).toBe(text);
+};
+
+describe('Deposit View', () => {
+  let browser;
+  let page;
+
+  const principalId = 'principalId';
+  const accountId = 'accountId';
+
+  const principalIdSubtitle = "Use when receiving from Plug accounts or other apps that support sending directly to Principal ID's.";
+  const accountIdSubtitle = "Use when receiving from exchanges or other apps that only support sending to Account ID's.";
+
+  beforeAll(async () => {
+    browser = await setupChrome();
+
+    // Importing and unlocking the accoun
+    page = await utils.createNewPage(browser);
+
+    await optionsPageUtils.importAccount(page, secrets.seedphrase, secrets.password);
+    await optionsPageUtils.unlock(page, secrets.password);
+
+    await page.close();
+  });
+
+  beforeEach(async () => {
+    page = await utils.createNewPage(browser);
+    await page.goto(chromeData.popupUrl);
+    await depositViewButtonClick(page);
+  });
+
+  afterEach(async () => {
```

Comment:
I don't think we need an `afterEach` here because there's only one `test`, what do you think?

Reply:
Same with beforeEach